core/debug/debug.c

```diff
@@ -4,7 +4,7 @@
 
 volatile bool in_debugger = false;
 
-CEMU_INLINE uint8_t debug_port_read_byte(const uint32_t addr) {
```

Comment:
This should not be removed, `CEMU_INLINE` is supposed to expand correctly depending on the compiler.